index.html

```diff
@@ -200,25 +207,35 @@ <h5>Skills</h5>
             </div>
         </section>
     </main>
+
     <footer class="footer">
-        <div class="footer-bg container">
-            <div class="contact-heading">
-                <h2>Contact me</h2>
-                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-            </div>
-            
-            <form action="https://formspree.io/f/xpznvdkn" method="post" class="contact-form">
-                <div class="form-input">
-                    <input type="text" required name="user_name" placeholder="Please enter your name" maxlength="30"/> 
-                </div>
-                <div class="form-input">
-                    <input type="email" required name="@gmail.com" placeholder="Please enter your email"/> 
-                </div>
-                <div class="form-input">
-                    <textarea required name="message" placeholder="Write your message here" maxlength="500"></textarea>
+        <div class="footer-color">
+            <div class="footer-bg">
+
+                <div class="container">
+                    <div class="contact-heading">
+                        <h2>Contact me</h2>
+                        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+                    </div>
+                    
+                    <form action="https://formspree.io/f/xpznvdkn" method="post" class="contact-form">
+                        <div class="form-input">
+                            <input type="text" required name="user_name" placeholder="Please enter your name" maxlength="30"/> 
+                        </div>
+                        <div class="form-input">
+                            <input type="email" required name="@gmail.com" placeholder="Please enter your email"/> 
+                        </div>
+                        <div class="form-input">
+                            <textarea required name="message" placeholder="Write your message here" maxlength="500"></textarea>
+                        </div>
+                        <div class="contact-btn">
+                            <button class="btn">Get It Touch</button>
+                        </div>
+                        
+                    </form>
```

Comment:
- Good job on the work done so far, however this section of your portfolio has some minor differences from what is in the design template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your design:
-- Please note the marked section in the image of the Figma design I attached below, I suggest you implement the missing design 🙏 
<img width="949" alt="yhryrt" src="https://user-images.githubusercontent.com/73607512/206399094-ed8c88a8-4845-43fd-95e7-c0ebead2919f.PNG">

Figma Design:
<img width="545" alt="rynyth" src="https://user-images.githubusercontent.com/73607512/206399288-65f23f62-7f24-4d25-8edb-c13883d9e5e2.PNG">

Reply:
Thank you, dunno how that fell off, I've just made the changes